pdbx/containers.py

```diff
@@ -355,8 +371,8 @@ def get_row(self, index):
     def remove_row(self, index):
         """Remove specified row."""
         try:
-            #if ((index >= 0) and (index < len(self._row_list))):
-            if (index >= 0 and self._row_list):
+            # if ((index >= 0) and (index < len(self._row_list))):
```

Comment:
remove this line?

Reply:
Yes, that is reasonable because it is written correctly below it. It looks like a remnant from trying to port it to Python3.